.github/workflows/build-and-test.yml

```diff
@@ -194,6 +195,79 @@ jobs:
         run: npm ci --build-from-source
         working-directory: bindings/nodejs
 
+  swift-bindings:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest]
+    
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+
+      # Temporary fix for "ValueError: invalid mode: 'rU' while trying to load binding.gyp"
+      # See <PR link>
+      # This can be removed when "prebuild" updates "node-gyp"
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```

Comment:
This is just for the Node.js bindings (because of `node-gyp`) so I think it can be removed for Swift